src/coreComponents/physicsSolvers/wavePropagation/ElasticVTIWaveEquationSEMKernel.hpp

```diff
@@ -0,0 +1,292 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file ElasticVTIWaveEquationSEMKernel.hpp
+ */
+
+#ifndef GEOS_PHYSICSSOLVERS_WAVEPROPAGATION_ElasticVTIWaveEquationSEMKERNEL_HPP_
+#define GEOS_PHYSICSSOLVERS_WAVEPROPAGATION_ElasticVTIWaveEquationSEMKERNEL_HPP_
+
+#include "finiteElement/kernelInterface/KernelBase.hpp"
+#include "WaveSolverUtils.hpp"
+#include "ElasticFields.hpp"
+
+namespace geos
+{
+using namespace fields;
+/// Namespace to contain the elastic wave kernels.
+namespace ElasticVTIWaveEquationSEMKernels
```

Comment:
Same here. This class should inherit from the base Elastic class.
you just need to override:
- the StackVariables class
- the setup() method
- the quadraturePointKernel() method

for example, template the base ElasticWaveEquationSEMKernel class:
```
class StackVariables;
template< typename SUBREGION_TYPE,
          typename CONSTITUTIVE_TYPE,
          typename FE_TYPE,
          typename STACK_VARS = StackVariables >
class ExplicitElasticVTISEM : public finiteElement::KernelBase< SUBREGION_TYPE, CONSTITUTIVE_TYPE, FE_TYPE, 1, 1 >

void setup( localIndex const k,
              STACK_VARS & stack ) const

etc...
```
and then in the VTI class
```
class StackVariables;
template< typename SUBREGION_TYPE,
          typename CONSTITUTIVE_TYPE,
          typename FE_TYPE >
class ExplicitElasticVTISEM : public ExplicitElasticSEM< SUBREGION_TYPE, CONSTITUTIVE_TYPE, FE_TYPE, StackVariables >
```